### PR TITLE
added SECURITY.md

### PR DESCRIPTION
**Pull request by gary-fallon**
## Description
Need to add SECURITY.md to inform security researchers on how to report security vulnerabilities.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#34726

## Testing done
N/A

## Screenshots
N/A

## Acceptance criteria
- [x] SECURITY.md is written and approved by Platform Security.

## Definition of done
- [x] Information on how to perform responsible disclosure included.
- [x] Description of boundary included in the document.
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Security Policy
+
+The Department of Veterans Affairs participates in a Responsible Vulnerability Disclosure Program through Bugcrowd. Please report any security-related issues using our [Bugcrowd program](https://bugcrowd.com/va-vdp).
+
+If your Bugcrowd submission is related to an open GH issue, please include that issue ID in the description field of your disclosure. 
+
+But, please do not open new security-related issues. If you are a developer and have concerns, you can reach out on our internal Slack channel #platform-security.
```
Comment (jjulian) [inline SECURITY.md:7]: Can we link to the slack workspace?

Comment (gary-fallon) [reply SECURITY.md]: Added.

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Security Policy
+
+The Department of Veterans Affairs participates in a Responsible Vulnerability Disclosure Program through Bugcrowd. Please report any security-related issues using our [Bugcrowd program](https://bugcrowd.com/va-vdp).
+
+If your Bugcrowd submission is related to an open GH issue, please include that issue ID in the description field of your disclosure. 
+
+But, please do not open new security-related issues. If you are a developer and have concerns, you can reach out on our internal Slack channel [#platform-security](https://dsva.slack.com/archives/C01CJV0L9PS).
```
Comment (mchelen-gov) [inline SECURITY.md:7]:
```suggestion
Please do not create security related issues in Github. If you are a developer and have concerns, you can reach out on our internal Slack channel [#platform-security](https://dsva.slack.com/archives/C01CJV0L9PS).
```

**Review by td-usds — APPROVED** [review]

**Comment by romizo** [comment]
👌

**Pull request closed without merging**